pkg/aws/cluster/validate_cluster_test.go

```diff
@@ -33,23 +33,39 @@ func TestValidateReleaseVersion(t *testing.T) {
 			name: "case 1",
 			ctx:  context.Background(),
 
-			newReleaseVersion: "1.0.0",
+			newReleaseVersion: "100.1.0",
 			valid:             true,
 		},
 		{
 			// version changed to deprecated release
 			name: "case 2",
 			ctx:  context.Background(),
 
-			newReleaseVersion: "2.0.0",
+			newReleaseVersion: "100.2.0",
 			valid:             false,
 		},
 		{
 			// version changed to invalid release
 			name: "case 3",
 			ctx:  context.Background(),
 
-			newReleaseVersion: "3.0.0",
+			newReleaseVersion: "100.3.0",
+			valid:             false,
+		},
+		{
+			// version changed with major skip
+			name: "case 4",
+			ctx:  context.Background(),
+
+			newReleaseVersion: "102.0.0",
+			valid:             false,
+		},
```

Comment:
With this we see that the test is not very informative. Such a table driven test would better inform about input and output if all inputs would be covered directly. I am wondering what the old release version is now.

Reply:
Since the old release version is the same for all tests, I did not include it. But if you feel like it adds a value I can include it.

Reply:
For you who wrote the tests that is fine. Though you work in a distributed team and nobody looking at the tests now knows which version is worked with in comparison. I just wanted to point out that in general you want to be as clear as possible in these cases so that you see at a glance what is input and what is output. Then a test is easy to understand and maintain.

Reply:
I changed it from using the default version to choosing the old release version explicitly.